Training_25/Program.cs

```diff
@@ -3,11 +3,66 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Assignment 2: NUMBER TO WORDS AND ROMAN NUMERALS CONVERTER.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training_25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      WriteLine ("Enter a number: ");
+      int num;
+      while (!int.TryParse (ReadLine (), out num))
+         WriteLine ("Invalid input. Please enter a valid number: ");
+      string? userChoice;
+      while (true) {
+         WriteLine ($"Input: {num}\nConvert {num} to Roman or Word? (Enter roman/word)");
+         userChoice = ReadLine ()?.ToLower ().Trim ();
+         switch (userChoice) {
+            case "roman": WriteLine (ConvertToRoman (num)); break;
+            case "word": WriteLine (ConvertToWords (num)); break;
+            default: WriteLine ("Invalid choice! Please enter 'roman' or 'word'."); continue;
+         }
+         break;
+      }
+      Console.WriteLine ("Press any key to exit..."); Console.ReadKey ();

```

Comment:
```suggestion
      WriteLine ("Press any key to exit..."); ReadKey ();
```
Since you added a static using statement for System.Console, you can directly call the method